text/0750-feature-flag-advisor.md

```diff
@@ -0,0 +1,343 @@
+      * @aws-cdk/...                              true                         - <unset>
+      Setting your unconfigured flags to the default values...
+
+    > cdk flags "@aws-cdk/aws-cloudfront:defaultSecurityPolicyTLSv1.2_2021" --set
+      What value would you like to change "@aws-cdk/aws-cloudfront:defaultSecurityPolicyTLSv1.2_2021" to?
+      > true
+        false
+```
+
+## README
+
+The new CLI command `cdk flags` shows you a report of your feature flag configurations that differ from our recommended states. It
+compares your current feature flag configurations and our recommended states. When you run `cdk flags`, a feature flags report is displayed
+to you in the following format: flags that do not match the recommended values and then flags that are not configured by the user at all.
+There will be a Menu with options for the user to configure their feature flags.
+
+```
+> cdk flags 
+    Feature Flag                              Recommended Value            User Value
+  * @aws-cdk/...                              true                         false
+  * @aws-cdk/...                              true                         false
+  * @aws-cdk/...                              true                         - <unset>
+  
+  How would you like to proceed?
+   >  Set all flags to recommended values 
+      Set unconfigured flags to recommended values
+      Reset unconfigured flags to defaults (no impact)
+      Modify individual flag 
+```
+
+### Expected Use Cases
+
+#### View information about your feature flags
+
+To view a report of all flags, users can run `cdk flags --all`. This will display a report of every feature flag in the following
+order: feature flags with states that match our recommendations, feature flags with states that do not match our recommended values, and
+lastly, unconfigured feature flags.
+
+```
+> cdk flags --all
+    Feature Flag                              Recommended Value            User Value
+    @aws-cdk/...                              true                         true
+  * @aws-cdk/...                              true                         false
+  * @aws-cdk/...                              true                         false
+  * @aws-cdk/...                              true                         - <unset>
+  
+  How would you like to proceed?
+   >  Set all flags to recommended values
+      Set all unconfigured flags to recommended values
+      Reset all unconfigured flags to defaults (no impact)
+      Modify individual flag
+```
+
+To view information about a specific flag, simply run `cdk flags "#FLAGNAME#"`.
+
+```
+> cdk flags "@aws-cdk/aws-cloudfront:defaultSecurityPolicyTLSv1.2_2021"
+    Feature Flag                                                        Recommended Value            User Value
+    "@aws-cdk/aws-cloudfront:defaultSecurityPolicyTLSv1.2_2021"         true                         true
+```
```

Comment:
At this point I would show the full explanation. So I think the layout should be different.

Think in terms of a UI as a list + detail pane. We're now past the list, and moved into the detail pane.